apps/docs/content/guides/platform/manage-your-usage/storage-image-transformations.mdx

```diff
@@ -0,0 +1,83 @@
+| Number Of Origin Images | Number Of Packages Billed | Costs |
+|-------------------------|---------------------------|-------|
+| 4                       | 1                         | $2    |
+| 5                       | 1                         | $2    |
+| 6                       | 2                         | $4    |
+
+### Usage on your invoice
+Storage Image Transformations usage is shown as "Storage Image Transformations" on your invoice.
+
+## Pricing
+$5 per 1000 origin images. You are only charged for usage exceeding your subscription plan's quota.
+
+<Admonition type="note">
+  The count resets at the start of each billing cycle.
+</Admonition>
+
+| Plan       | Quota             | Over-Usage  |
+|------------|-------------------|-------------|
+| Pro        | 100 origin images | $5 per 1000 |
+| Team       | 100 origin images | $5 per 1000 |
+| Enterprise | Custom            | Custom      |
+
+## Billing examples
+### Within quota
+The organization's number of origin images for the billing cycle is within the quota, so no charges apply.
+
+| Line Item             | Units     | Costs   |
+|-----------------------|-----------|---------|
+| Pro Plan              | 1         | $25     |
+| Compute Hours Micro   | 744 hours | $10     |
+| Image Transformations | 74 origin images       | $0      |
+| **Subtotal**          |           | **$35** |
+| Compute Credits       |           | -$10    |
+| **Total**             |           | **$25** |
+
+### Exceeding quota
+The organization's number of origin images for the billing cycle exceeds the quota by 750, incurring charges for this additional usage.
+
+| Line Item             | Units             | Costs   |
+|-----------------------|-------------------|---------|
+| Pro Plan              | 1                 | $25     |
+| Compute Hours Micro   | 744 hours         | $10     |
+| Image Transformations | 850 origin images | $5      |
+| **Subtotal**          |                   | **$40** |
+| Compute Credits       |                   | -$10    |
+| **Total**             |                   | **$30** |
+
+## View usage
+You can view Storage Image Transformations usage on the [organization's usage page](https://supabase.com/dashboard/org/_/usage). The page shows the usage of all projects by default. To view the usage for a specific project, select it from the dropdown. You can also select a different time period.
+<Image
+  alt="Usage page navigation bar"
+  src={{
+    light: '/docs/img/guides/platform/usage-navbar--light.png',
+    dark: '/docs/img/guides/platform/usage-navbar--dark.png',
+  }}
+  zoomable
+/>
+
+In the Storage Image Transformations section, you can see how many origin images were transformed during the selected time period.
+<Image
+  alt="Usage page Storage Image Transformations section"
+  src={{
+    light: '/docs/img/guides/platform/usage-image-transformations--light.png',
+    dark: '/docs/img/guides/platform/usage-image-transformations--dark.png',
+  }}
+  zoomable
+/>
```

Comment:
This is missing info on how to reduce usage - i.e. use external caching, save the transformed image and serve it instead of always transforming on the fly, if you don't need different transformations - save the image in the correct size already and serve it, ...

Reply:
Added examples